docs/install/amdgpu-install.rst

```diff
@@ -22,7 +22,9 @@ Installation
 
 Installation of ``amdgpu-install`` differs slightly depending on the OS and its package manager.
 
-Make sure that the :doc:`/install/prerequisites` are met before installing.
+.. note::
```

Comment:
```suggestion
.. important::
```

Do you want to try changing from "note" to "important"?  Do we want it to standout more?

Reply:
Yea, I agree. Wasn't aware of the important block